src/actors/resolver.rs

```diff
@@ -121,25 +121,46 @@ impl Message for ConnectAddr {
     type Result = Result<TcpStream, ResolverError>;
 }
 
-#[derive(Debug, Display)]
 pub enum ResolverError {
     /// Failed to resolve the hostname
-    #[display(fmt = "Failed resolving hostname: {}", _0)]
     Resolver(String),
 
     /// Address is invalid
-    #[display(fmt = "Invalid input: {}", _0)]
     InvalidInput(&'static str),
 
     /// Connecting took too long
-    #[display(fmt = "Timeout out while establishing connection")]
     Timeout,
 
     /// Connection io error
-    #[display(fmt = "{}", _0)]
     IoError(io::Error),
 }
 
+impl fmt::Debug for ResolverError {
+    fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Self::Resolver(s) => fmt.debug_tuple("Resolver").field(s).finish(),
+            Self::InvalidInput(s) => fmt.debug_tuple("InvalidInput").field(s).finish(),
+            Self::Timeout => fmt.debug_tuple("Timeout").finish(),
+            Self::IoError(e) => fmt.debug_tuple("IoError").field(e).finish(),
+        }
+    }
+}
```

Comment:
what's up with built-in Debug derive?

Reply:
Other error types use manual Debug so I think it could be in consistent.